telemetry/specs/INT.mdk

```diff
@@ -1018,8 +1018,12 @@ and it assumes INT over TCP as the encapsulation protocol.
 **This program is work in progress.**
```

Comment:
A few lines up it says "The program is written in P4_16 on [v1model architecture]"

Reply:
Done.